ui/app/src/views/ViewDashboard.tsx

```diff
@@ -156,9 +156,9 @@ function ViewDashboard() {
                   dashboardProject={data.metadata.project}
                 />
               }
```

Comment:
See code in `DashboardApp` for a bit of clarity on the why of switching from a component to component props here.